### PR TITLE
GETOK - Integrate the CSST POSTMAN collection and instructions.

### PR DESCRIPTION
**Pull request by TimCsaky**
# Description

Supports ticket: [SHOWCASE-1341](https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-1341)

I'm not sure if the screenshots on the Jira Ticket were approved by all.
I've changed a coupl epof the bits of wording but not sure if they are an improvement.
Let me know if you prefer the text in the screenshots.

## Types of changes

Documentation (non-breaking change with enhancements to documentation)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/frontend/src/components/RequestForm.vue [hunk 1]
```diff
@@ -151,7 +151,7 @@ export default {
           v.length <= FieldValidations.ACRONYM_MAX_LENGTH ||
           `Acronym must be ${FieldValidations.ACRONYM_MAX_LENGTH} characters or less`,
         v =>
-          /^(?:[A-Z]{2,}[_]?)+[A-Z]{1,}$/g.test(v) ||
+          /^(?:[A-Z]{1,}[_]?)+[A-Z]{1,}$/g.test(v) ||
```
Comment (TimCsaky) [inline app/frontend/src/components/RequestForm.vue:154]: This rule wasnt inline with the validation message.
ABC_D_EFG wasnt allowed
ABC_DE_FGH was allowed
seems a single character between underscores wasnt allowed.
I've changed it so it is allowed. Let me know if that should have been a requirement.
If so, i wonder if it needs to be mentioned in the validation tooltip

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-k8vopl-tools.pathfinder.gov.bc.ca/dashboard?id=nr-get-token-pr-180

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-k8vopl-tools.pathfinder.gov.bc.ca/dashboard?id=nr-get-token-pr-180

File: app/frontend/src/components/Resources.vue [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+                <p>We do recommend that you read the API documentation before running the collection.</p>
+                <ul>
+                  <li class="my-2">
+                    Common Hosted Email Service (
+                    <a href="https://ches-master-9f0fbe-prod.pathfinder.gov.bc.ca/api/v1/docs">CHES</a>).
+                  </li>
+                  <li class="my-2">
+                    Common Document Generation Service (
+                    <a
+                      href="https://cdogs-master-idcqvl-prod.pathfinder.gov.bc.ca/api/v2/docs#tag/DocGen"
```
Comment (jujaga) [inline app/frontend/src/components/Resources.vue:53]: This link shouldn't have the #tag/DocGen part after it

File: app/frontend/src/components/Resources.vue [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+              </td>
+            </tr>
+            <tr>
+              <td>API Documentation</td>
+              <td>
+                <p>We do recommend that you read the API documentation before running the collection.</p>
+                <ul>
+                  <li class="my-2">
+                    Common Hosted Email Service (
+                    <a href="https://ches-master-9f0fbe-prod.pathfinder.gov.bc.ca/api/v1/docs">CHES</a>).
```
Comment (jujaga) [inline app/frontend/src/components/Resources.vue:48]: There should be a space before the closing parentheses to make it look more balanced as compared to the opening parenthesis. Also affects line 54.

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-k8vopl-tools.pathfinder.gov.bc.ca/dashboard?id=nr-get-token-pr-180

File: app/frontend/src/components/apiAccess/Step4.vue [hunk 4]
```diff
@@ -4,10 +4,8 @@
       <v-icon large color="green darken-2">check_circle</v-icon>API Documentation and Usage
     </h2>
     <p class="mb-12">
-      This service client can be used to make REST API calls and an example Postman collection is provided.
-      <br />Fetch a new token using the service client and password against the token endpoint (see postman collection) and use that token in your bearer header.
+      Your service client can make REST calls to a Common Services API. Please view the <router-link :to="{ name: 'Documentation' }">"Documentation".</router-link> page to learn more about these API's, get the full specifications and learn how to test them with Postman.
```
Comment (jujaga) [inline app/frontend/src/components/apiAccess/Step4.vue]: Your service client can make REST calls to the Common Service APIs. Please view the Documentation to learn more about these APIs, their specifications, and how to test them with Postman.

File: app/frontend/src/components/RequestForm.vue (hunk 1 above)
Comment (jujaga) [reply app/frontend/src/components/RequestForm.vue:154]: Original regex requirement here: https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-65

File: app/frontend/src/components/Resources.vue [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+                  </li>
+                </ul>
+              </td>
+            </tr>
+            <tr>
+              <td>POSTMAN Collection</td>
+              <td>
+                <p>Use the Common Service Postman Collection to verify your service client credentials and see the APIs in action.</p>
+                <ul>
+                  <li class="my-2"><router-link :to="{ name: 'RequestAccount' }">Get your service client</router-link>.
```
Comment (jujaga) [inline app/frontend/src/components/Resources.vue:32]: I'd suggest only "service client" be as part of the hyperlink. Looks a bit strange to have an entire bullet point be a link when later points don't do the same thing.

Comment (TimCsaky) [reply app/frontend/src/components/Resources.vue]: sure

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-k8vopl-tools.pathfinder.gov.bc.ca/dashboard?id=nr-get-token-pr-180

**Pull request merged**